IFC4x3/Properties/w/Weathering_3VudkS3GP7whCc9fnu5Tf9/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Weathering_3VudkS3GP7whCc9fnu5Tf9" Name="Weathering" UniqueId="dfe27b9c-0d06-47ea-b326-269c7815da49" PrimaryDataType="IfcLabel" />
```

Comment:
If it is supposed to be a description, maybe consider `IfcText`.

Reply:
Ping @j-weil

Reply:
Weathering should reference ISO 14689:2017(E) and be provided according to the table below:
![image](https://github.com/bSI-InfraRoom/IFC-Specification/assets/49076658/56564911-d27d-4860-b876-43fa4c51558d)
Column "Term" as predefined enumerations and adding the ref to ISO standard would be perfect